crates/host/Cargo.toml

```diff
@@ -7,8 +7,8 @@ authors = ["thedavidmeister", "thedavidmeister@gmail.com"]
 edition = "2021"
 
 [dependencies]
-wasmer = "=4.3.1"
-wasmer-middlewares = "=4.3.1"
+wasmer = { git = "https://github.com/wasmerio/wasmer.git", branch = "wamr", package = "wasmer", default-features = false}
+wasmer-middlewares = { git = "https://github.com/wasmerio/wasmer.git", branch = "wamr", package = "wasmer-middlewares", optional = true }
```

Comment:
First off, congrats! I'm excited you got it working :rocket: 

What's the plan for publishing this? We can't publish with git references. Are we waiting for upstream to publish? Are we going to publish a fork in the mean time?

Reply:
It sounds like wasmer team wants to be a lot more confident before merging it into a release. Here's their explanation:

![image](https://github.com/user-attachments/assets/e5d07f13-77fe-44ef-88ff-ac6306518774)

Sounds like we'll need to publish a fork for now if we can't work off git references?

Reply:
I presume, then, that this branch has other untested changes included even for the non-wamr mode. Did the api change much/at all? It might be preferable then to use the official release for the `sys` feature and our fork release for the `wamr` feature.

Reply:
The branch only includes changes for adding the wamr mode -- the api didn't change at all. Yes that makes sense to fork release only for the wamr feature. I'll do that.